src/services/example_data/example_mappings.ts

```diff
@@ -0,0 +1,10 @@
+import { type SensorLocation } from '@/types/models/SensorLocation';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
```suggestion
```
This rule was disabled on the `master` branch. Rebase your branch (or merge changes from `master`) ans you will be able to remove this comment.